Sources/Response.swift

```diff
@@ -24,33 +26,36 @@ public protocol ResponseRepresentable {
 public protocol ResponseConvertible: ResponseInitializable, ResponseRepresentable {}
 
 extension Response {
-    public init(status: Status = .ok, headers: Headers = [:], body: Data = []) {
+    public init(status: Status = .ok, headers: [CaseInsensitiveString: String] = [:], cookies: Set<String>, body: Data = []) {
         self.init(
             version: Version(major: 1, minor: 1),
             status: status,
             headers: headers,
+            cookies: cookies,
             body: .buffer(body)
         )
 
-        self.headers["Content-Length"] += body.count.description
+        self.headers["Content-Length"] = body.count.description
     }
 
-    public init(status: Status = .ok, headers: Headers = [:], body: Stream) {
+    public init(status: Status = .ok, headers: [CaseInsensitiveString: String] = [:], cookies: Set<String>, body: Stream) {
```

Comment:
I think we can, but there wouldn't be a problem actually. Cause if you do:

```swift
public init(status: Status = .ok, headers: [CaseInsensitiveString: String] = [:], cookies: Set<Cookie>, body: Stream)
```

The compiler will be able to disambiguate because of the type.